### PR TITLE
adjust javadoc links in scripting page

### PR DESCRIPTION
**Pull request by zoollcar**
Replace fixed url with the javadocref function

### CODE REVIEW

File: content/user-guide/process-engine/scripting.md [hunk 1]
```diff
@@ -338,9 +338,9 @@ i.e., `$sum`)
 
 There are also special variables:
 
-1. `execution`, which is always available if the script is executed in an execution scope (e.g., in a script task) ([`DelegateExecution`](https://docs.camunda.org/javadoc/camunda-bpm-platform/7.7/org/camunda/bpm/engine/delegate/DelegateExecution.html)).
-1. `task`, which is available if the script is executed in a task scope (e.g., a task listener) ([`DelegateTask`](https://docs.camunda.org/javadoc/camunda-bpm-platform/7.7/org/camunda/bpm/engine/delegate/DelegateTask.html)).
-1. `connector`, which is available if the script is executed in a connector variable scope (e.g., outputParameter of a camunda:connector) ([`ConnectorVariableScope`](https://docs.camunda.org/javadoc/camunda-bpm-platform/7.7/org/camunda/connect/plugin/impl/ConnectorVariableScope.html)).
+1. `execution`, which is always available if the script is executed in an execution scope (e.g., in a script task) ({< javadocref page="?org/camunda/bpm/engine/delegate/DelegateExecution.html" text="DelegateExecution" >}}).
```
Comment (yanavasileva) [inline content/user-guide/process-engine/scripting.md:341]: There's a missing bracket that breaks the link creation:
```suggestion
1. `execution`, which is always available if the script is executed in an execution scope (e.g., in a script task) ({{< javadocref page="?org/camunda/bpm/engine/delegate/DelegateExecution.html" text="DelegateExecution" >}}).
```

File: content/user-guide/process-engine/scripting.md [hunk 2]
```diff
@@ -363,8 +363,9 @@ task = execution.getProcessEngineServices().getTaskService()
 
 Camunda's Java API provides access to Camunda's process engine services; these services can be accessed using Scripts:
 
-[Process Engine Services](https://docs.camunda.org/javadoc/camunda-bpm-platform/7.3/org/camunda/bpm/engine/ProcessEngineServices.html)
-[Public Java API of Camunda Platform Engine](https://docs.camunda.org/javadoc/camunda-bpm-platform/7.7/org/camunda/bpm/engine/package-summary.html)
+{{< javadocref page="?org/camunda/bpm/engine/ProcessEngineServices.html" text="Process Engine Services" >}}
```
Comment (yanavasileva) [inline content/user-guide/process-engine/scripting.md:366]: Let's add a line break for readability: 
```suggestion
{{< javadocref page="?org/camunda/bpm/engine/ProcessEngineServices.html" text="Process Engine Services" >}} \
```

**Review by yanavasileva — CHANGES_REQUESTED** [review]
Hi @zoollcar,

Thank you for your contribution, nice catch.

Please have a look at my suggestions below, other than that all looks good.

Best regards,
Yana

**Comment by yanavasileva** [comment]
In addition, please have a look at our Contribution guide and commit message conventions for your future reference:
https://github.com/camunda/camunda-bpm-platform/blob/master/CONTRIBUTING.md#commit-message-conventions

**Comment by zoollcar** [comment]
> In addition, please have a look at our Contribution guide and commit message conventions for your future reference:
> https://github.com/camunda/camunda-bpm-platform/blob/master/CONTRIBUTING.md#commit-message-conventions

Ok, I get it.

**Comment by yanavasileva** [comment]
@zoollcar, thank you for applying the requested changes and once again thanks for made the contribution. I will merge the adjustments.

**Pull request merged**